docs/apache-airflow-providers-amazon/operators/eks.rst

```diff
@@ -0,0 +1,265 @@
+
+Defining tasks
+""""""""""""""
+
+In the following code we retrieve details for a given Amazon EKS nodegroup.
+
+.. exampleinclude:: /../../airflow/providers/amazon/aws/example_dags/example_eks_create_nodegroup.py
+    :language: python
+    :start-after: [START howto_operator_eks_describe_nodegroup]
+    :end-before: [END howto_operator_eks_describe_nodegroup]
+
+
+In the following code we list all Amazon EKS Nodegroups in a given EKS Cluster.
+
+.. exampleinclude:: /../../airflow/providers/amazon/aws/example_dags/example_eks_create_nodegroup.py
+    :language: python
+    :start-after: [START howto_operator_eks_list_nodegroup]
+    :end-before: [END howto_operator_eks_list_nodegroup]
+
+
+.. _howto/operator:EKSDeleteNodegroupOperator:
+
+Deleting Amazon EKS Managed Nodegroups
+--------------------------------------
+
+Defining tasks
+""""""""""""""
+
+In the following code we delete an Amazon EKS nodegroup.
+
+.. exampleinclude:: /../../airflow/providers/amazon/aws/example_dags/example_eks_create_nodegroup.py
+    :language: python
+    :start-after: [START howto_operator_eks_delete_nodegroup]
+    :end-before: [END howto_operator_eks_delete_nodegroup]
+
+
+Creating Amazon EKS Clusters and Node Groups Together
+------------------------------------------------------
+
+Purpose
+"""""""
+
+This example dag ``example_eks_create_stack.py`` demonstrates using
+``EKSCreateClusterOperator`` to create an Amazon EKS cluster and underlying
+Amazon EKS node group in one command.  ``EKSDescribeClustersOperator`` and
+``EKSDescribeNodegroupsOperator`` verify creation, then ``EKSDeleteClusterOperator``
+deletes all created resources.
+
+Prerequisites
+"""""""""""""
+
+  ``ec2.amazon.aws.com`` must be in the Trusted Relationships
+  ``eks.amazonaws.com`` must be added to the Trusted Relationships
+  ``AmazonEC2ContainerRegistryReadOnly`` IAM Policy must be attached
+  ``AmazonEKSClusterPolicy`` IAM Policy must be attached
+  ``AmazonEKSWorkerNodePolicy`` IAM Policy must be attached
+
+Defining tasks
+""""""""""""""
+
+In the following code we create a new Amazon EKS cluster and node group, verify creation,
+then delete both resources.
+
+.. exampleinclude:: /../../airflow/providers/amazon/aws/example_dags/example_eks_create_cluster_with_nodegroup.py
+    :language: python
+    :start-after: [START howto_operator_eks_create_cluster_with_compute]
+    :end-before: [END howto_operator_eks_create_cluster_with_compute]
+
+
+.. _howto/operator:EKSPodOperator:
+
+Perform a Task on an Amazon EKS Cluster
+---------------------------------------
+
+Purpose
+"""""""
+
+This example dag ``example_eks_pod_operator.py`` demonstrates using
+``EKSStartPodOperator`` to perform a command on Amazon EKS cluster.
+
+Prerequisites
+"""""""""""""
+
+  1. An Amazon EKS Cluster with underlying compute infrastructure.
+  2. The AWS CLI version 2 must be installed on the worker.
+
+  see: https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html
```

Comment:
We should use descriptive links. See: https://developers.google.com/style/link-text

Reply:
Link dropped, but will keep this in mind in the future.